packages/swarmplot/index.d.ts

```diff
@@ -47,7 +47,7 @@ declare module '@nivo/swarmplot' {
         Axes = 'axes',
         Nodes = 'nodes',
         Mesh = 'mesh',
-        Annotations = 'annotations'
+        Annotations = 'annotations',
```

Comment:
This got fixed by the linter checks, so I let it in with my comment.

Reply:
Thanks, I think this is what is failing the build.